evap/evaluation/models.py

```diff
@@ -1738,6 +1738,13 @@ class Meta:
     def save(self, *args, **kwargs):
         # This is not guaranteed to be called on every insert. For example, the importers use bulk insertion.
 
+        from evap.evaluation.auth import password_login_is_active
```

Comment:
@niklasmohrin this is a bit annoying: When defining the helper methods in auth.py, we can not include `auth` in `models` at the beginning of the file because `auth` already includes `models` at the top of the file. I guess we want to move the method to `tools`, then? We could also implement it in `auth.py` before importing the models, so the file would have generic imports, then generic definitions, then model imports, and then model-specific definitions.

I will have to look through the django documentation, maybe they have some standard structuring for this...

Reply:
Moving to `tools`, while annoying and unsatisfying, seems fair to me. If Django suggests something else, I am also fine with that

Reply:
Would you prefer the inline import in the model method, or to have the methods moved to tools? Not quite a fan of both options, but it probably also doesn't really matter. Moving to `tools` would ensure that there's no performance penalty for importing here, but realistically the import should only make a difference the first time we come across the imports.

Reply:
I think I slightly prefer moving the functions to `tools`